tests/test_lab/test_detectors.py

```diff
@@ -319,6 +319,35 @@ def test_sampling_mean_and_var(
         var = results.var(axis=0)
         assert np.allclose(var[0], var_expected, atol=self.std_10, rtol=0)
 
+    @pytest.mark.parametrize(
+        "state, kwargs, mean_expected, var_expected",
+        [
+            (Vacuum, {"num_modes": 1}, 0.0, settings.HBAR / 2),
+            (Coherent, {"x": 2.0, "y": 0.5}, 2.0 * np.sqrt(2 * settings.HBAR), settings.HBAR / 2),
+            (SqueezedVacuum, {"r": 0.25, "phi": 0.0}, 0.0, 0.25 * settings.HBAR / 2),
+        ],
+    )
+    def test_sampling_mean_and_var_unnormalized_state(
+        self, state, kwargs, mean_expected, var_expected
+    ):
+        """Tests that the mean and variance estimates of many homodyne
+        measurements are in agreement with the expected values for the given (unnormalized) states
+        """
+        state = state(**kwargs)
+        state = State(dm=state.dm(cutoffs=[10]) / 3)
```

Comment:
Since this is the only difference with the test above (that I can detect), why not add a "normalization" argument to the function above, and then add to parametrize to check for both [1,3]?